app/views/content/blog/overcoming-challenges-to-become-a-teacher-autism.md

```diff
@@ -25,7 +25,7 @@ tags:
   - teacher training advisers
 ---
 
-*Felix offered to speak to Teacher Training Adviser, Jane Wilkinson, about the challenges faced by someone who lives with additional needs and their progress towards achieving the dream of becoming a teacher.*
+Felix offered to speak to Teacher Training Adviser, Jane Wilkinson, about the challenges faced by someone who lives with additional needs and their progress towards achieving the dream of becoming a teacher.
```

Comment:
Suggest moving this beneath the photo (but keeping as normal rather than italicised text)